main.py

```diff
@@ -8,12 +9,13 @@ def main():
     """
     year = "2021"
     cve_year_collection = CVEsCollection(year)
-    cve_dict = cve_year_collection.getCVE_Items()
-    cve_year_collection.getCveByIndex(0)
+    cve = cve_year_collection.getCveByIndex(51)
     print("Hello")
 
 
 
```

Comment:
CVEsCollection: You didn't change the file according to the talk we had last sunday. let's talk about it again :)